agent/agentssh/agentssh.go

```diff
@@ -411,9 +443,9 @@ func (s *Server) startPTYSession(logger slog.Logger, session ptySession, magicTy
 	}
 
 	if !isQuietLogin(s.fs, session.RawCommand()) {
-		manifest := s.Manifest.Load()
-		if manifest != nil {
-			err := showMOTD(s.fs, session, manifest.MOTDFile)
+		motdFile := s.config.MOTDFile()
+		if motdFile != "" {
+			err := showMOTD(s.fs, session, motdFile)
```

Comment:
Could `showMOTD` take no-action if motdFile is empty? simpler logic here

Reply:
Sure, good suggestion 👍🏻